### PR TITLE
fix_corba_bad_param_issue

### PR DESCRIPTION
**Pull request by ddliu365**
@cmastalli 
This is a pr to address [issue](https://github.com/loco-3d/crocoddyl/issues/896#issue-777084374) #896

### CODE REVIEW

**Comment by hrp2-14** [comment]
Hi ! This project doesn't usually accept pull requests on master. If this wasn't intentionnal, you can change the base branch of this pull request to devel (No need to close it for that). Best, a bot.

**Review by cmastalli — APPROVED** [review]
Thanks @ddliu365 for your contribution :)
Thanks @jmirabel for pointing out quickly the error :)

Looking forward for having more contributions from your part.
I apologize if this was not sorted out earlier.

File: bindings/python/crocoddyl/__init__.py [hunk 1]
```diff
@@ -310,13 +310,13 @@ def _setConeMu(self, coneName, mu):
...
-            self.robot.viewer.gui.deleteNode(coneGroup + "/lines", "")
-            self.robot.viewer.gui.deleteNode(coneGroup + "/cone", "")
-            self.robot.viewer.gui.deleteNode(coneGroup, "")
+            self.robot.viewer.gui.deleteNode(coneGroup + "/lines/0", True)
+            self.robot.viewer.gui.deleteNode(coneGroup + "/lines/1", True)
+            self.robot.viewer.gui.deleteNode(coneGroup + "/lines/2", True)
+            self.robot.viewer.gui.deleteNode(coneGroup + "/lines/3", True)
+            self.robot.viewer.gui.deleteNode(coneGroup + "/lines", True)
+            self.robot.viewer.gui.deleteNode(coneGroup + "/cone", True)
+            self.robot.viewer.gui.deleteNode(coneGroup, True)
```
Comment (jmirabel) [inline bindings/python/crocoddyl/__init__.py:319]: The boolean value toggle resistivity. As node *coneGroup* seems to be a group, calling `deleteNode(coneGroup, True)` automatically deletes child nodes so the last line would be sufficient here.

Comment (cmastalli) [reply bindings/python/crocoddyl/__init__.py:319]: What should I do in the previous lines? should I add `False` or do not include any argument?

Comment (jmirabel) [reply bindings/python/crocoddyl/__init__.py:319]: You can remove everything but the last line. `deleteNode(coneGroup, True)` will delete all nodes inside the group too.

This is not mandatory. You may prefer to keep the explicit deletion.

Comment (ddliu365) [reply bindings/python/crocoddyl/__init__.py:319]: I checked the animation in Gepetto, it is same with these codes and without theses codes. For clarity, I just deleted the first six lines in the new pr #898, but I am not sure which one is better.

Comment (cmastalli) [reply bindings/python/crocoddyl/__init__.py:319]: I will mark as resolved this discussion since the reviews are done in the PR.
Thanks guys :)

**Pull request merged**